pkgs/jovian-greeter/greeter.py

```diff
@@ -122,11 +125,15 @@ def _recv(self) -> Mapping[str, Any]:
 class Context:
     def __init__(self, user: str, home: Path):
         self.user = user
+        logging.debug("USER: {}".format(self.user))
```

Comment:
when logging variables in python, best practice is to use `%` interpolation.
```suggestion
        logging.debug("USER: %s", self.user)
```
see: https://docs.python.org/3/howto/logging.html#logging-variable-data